hips/draw/simple.py

```diff
@@ -1,7 +1,8 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """HiPS tile drawing -- simple method."""
-from typing import Tuple, List
 import numpy as np
+from typing import Tuple, List
+import matplotlib.pyplot as plt
```

Comment:
MPL is an optional dependency for the `hips` package. One should be able to draw and e.g. save a sky image without having it installed. If you put the MPL import here at the top of `simple.py`, that's not possible.

So instead, for optional dependencies, what we do is to put a "delayed" import on the first line of each function or method where you use it. That way, the import only happens if a user uses some functionality that requires the optional dependency.